docs/reverse_proxy.md

```diff
@@ -80,6 +80,11 @@ server {
         # Increase client_max_body_size to match max_upload_size defined in homeserver.yaml
         client_max_body_size 50M;
     }
+    location /.well-known/matrix/client {
+        return 200 '{"m.homeserver": {"base_url": "https://<matrix.example.com>"}}';
```

Comment:
we should probably make this match the example for Caddy, Apache and haproxy (ie, make it include `m.identity_server`)?